src/FSharpVSPowerTools.Core/UntypedAstUtils.fs

```diff
@@ -9,7 +9,7 @@ type internal ShortIdent = string
 type internal Idents = ShortIdent[]
 
 let internal longIdentToArray (longIdent: LongIdent): Idents =
-    longIdent |> List.map string |> List.toArray
+    Array.init longIdent.Length (fun i -> string longIdent.[i])
```

Comment:
I don't think this is desirable. `List.length` is `O(n)`. Combining with index access `.[i]`, the whole `Array.init` call would be of `O(n^2)` where n is the length of the list.

If we want to avoid allocation, changing to

```fsharp
Seq.map string |> Seq.toArray
```

would be enough